.travis.yml

```diff
@@ -25,10 +27,10 @@ install:
   - conda config --add channels mosdef
   - conda create -n test-environment python=$PYTHON_VERSION --file requirements.txt
   - source activate test-environment
-  - pip install -e .
+  - python setup.py install
```

Comment:
This really shouldn't be the issue here. The brew/rvm fixes were all that foyer needed, correct?